src/icrc1_auction_api.mo

```diff
@@ -497,4 +485,34 @@ actor class Icrc1AuctionAPI(trustedLedger_ : ?Principal, adminPrincipal_ : ?Prin
     stableAdminsMap := adminsMap.share();
   };
 
+  // A timer for consolidating backlog subaccounts
+  ignore Timer.recurringTimer<system>(
+    #seconds 60,
+    func() : async () {
+      for (asset in Vec.vals(assets)) {
+        await* asset.handler.trigger(1);
+      };
+    },
+  );
+
+  let AUCTION_INTERVAL_SECONDS : Nat64 = 86_400; // a day
+  private func remainingTime() : Nat = Nat64.toNat(AUCTION_INTERVAL_SECONDS - (Prim.time() / 1_000_000_000) % AUCTION_INTERVAL_SECONDS);
+
+  // run daily at 12:00 a.m. UTC
```

Comment:
Can we shift this by 12h to 12pm noon UTC?

Reply:
Done